components/Facility/index.js

```diff
@@ -26,39 +27,40 @@ export default function Facility({ facility }) {
           height={80}
           priority
         />
-        <StyledArticle>
-          <p>Kategorie: {category}</p>
-          <p>
+        <StyledList>
+          <ListItem>Kategorie: {category}</ListItem>
+          <ListItem>
             Adresse:
             <StyledLink href={link}> {adress}</StyledLink>
-          </p>
-          <p>Zielgruppe: {target}</p>
-          <p>Zugangsvoraussetzungen: {requirements}</p>
-        </StyledArticle>
+          </ListItem>
+          <ListItem>Zielgruppe: {target}</ListItem>
+          <ListItem>Zugangsvoraussetzungen: {requirements}</ListItem>
+        </StyledList>
       </StyledSection>
     </>
   );
 }
 const StyledImage = styled(Image)`
   object-fit: contain;
-  layout: responsive;
+  layout: "responsive";
```

Comment:
The layout property is no longer supported at Next 13. Regardless, `layout` is/was not a css property anyway. You may safely remove this line entirely.

Reply:
Alright. Thank you :-)